tests/test_utils.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+import pytest
+
+from xtractmime.utils import _is_mp4_signature, _is_webm_signature
+
+class TestUtils:
+
+    file = open("tests/files/foo.mp4","rb")
+    body_mp4 = file.read()
+    file.close()
```

Comment:
```suggestion
    with open("tests/files/foo.mp4", "rb") as fp:
        body_mp4 = fp.read()
```

There are a few other occurrences.